pages/searchpage.css

```diff
@@ -0,0 +1,25 @@
+
+
+.form-input{
```

Comment:
Format code using Prettifier to remove unneccesarry spaces

Reply:
Yes will keep in mind from now on